docs/source/x_ext.rst

```diff
@@ -570,13 +576,10 @@ then it shall issue no further memory request transactions for the same instruct
 be issued by a |coprocessor| for the same instruction (``id``). A sequence of memory request transactions therefore does not necessarily end with a transaction with ``last`` = 1.
```

Comment:
(This is not part of what the PR changed, but I want to comment on it.)

About the `last` signal. I cannot see that there is a rule to stop issuing memory requests (for the same id) after signaling `last=1`.
Do we want that rule, or do we want to allow more same-id requests after "last=1"?
(I suspect the pipeline would want to make some decisions based on "last", so maybe we want that rule?)

The spec currently says _"Normally a sequence of memory request transactions ends with a transaction that has last = 1"_.
And the rest of the paragraph hints towards what should happen after "last=1".
But it is not asserting it explicitly.
It also has the potentially misleading phrase _"A sequence of memory request transactions therefore does not necessarily end with a transaction with last = 1."_.

Reply:
>>Do we want that rule, or do we want to allow more same-id requests after "last=1"?

Agreed that we want a rule to forbid further memory requests with the same id after signaling last = 1. I will add that in this PR and I will see if I can rephrase the sentence that you state as maybe being misleading.